src/libs/actions/Report.ts

```diff
@@ -2737,7 +2737,7 @@ function buildNewReportOptimisticData(policy: OnyxEntry<Policy>, reportID: strin
         {
             onyxMethod: Onyx.METHOD.MERGE,
             key: `${ONYXKEYS.COLLECTION.REPORT}${parentReport?.reportID}`,
-            value: {lastVisibleActionCreated: optimisticReportPreview.created},
+            value: {lastVisibleActionCreated: optimisticReportPreview.created, hasOutstandingChildRequest: true},
```

Comment:
This should be true if and only if the policy harvesting is disabled - because if its enabled the report will be submitted automatically